src/ILCompiler.Compiler/src/Compiler/MetadataManager.cs

```diff
@@ -407,6 +407,24 @@ protected virtual void GetRuntimeMappingDependenciesDueToReflectability(ref Depe
             // mapping for a type.
         }
 
+        /// <summary>
+        /// This method is an extension point that can provide additional metadata-based dependencies to generated RuntimeMethodHandles.
+        /// </summary>
+        public virtual void GetDependenciesDueToLDToken(ref DependencyList dependencies, NodeFactory factory, MethodDesc method)
```

Comment:
Nit: I know we are not 100% consistent about ldtoken casing, but I do not think we are using `LDToken` anywhere.